pkg/controller/virtinformers.go

```diff
@@ -224,6 +229,22 @@ func (f *kubeInformerFactory) getInformer(key string, newFunc newSharedInformer)
 	return informer
 }
 
+func (f *kubeInformerFactory) Namespace() cache.SharedIndexInformer {
+	return f.getInformer("namespaceInformer", func() cache.SharedIndexInformer {
+		lw := cache.NewListWatchFromClient(f.clientSet.CoreV1().RESTClient(), "namespaces", k8sv1.NamespaceAll, fields.Everything())
```

Comment:
If you are patching the namespace, why would you care about all of its fields?
Can you not just select the namespace's labels?

Reply:
The `fields` selector allows us to filter resources we want to watch by different fields. For example, watch just namespaces with a certain label applied, or a namespace with a certain name. We will store the entire object anyway.

Reply:
I see, thanks for the explanation!